--- a/plinth/module_loader.py
+++ b/plinth/module_loader.py
@@ -146,7 +146,7 @@
     # Omit hidden files
     file_names = [file
                   for file in os.listdir(module_directory)
-                  if not file.startswith('.')]
+                  if not file.startswith('.') and '.dpkg' not in file]
 
     for file_name in file_names:
         full_file_name = os.path.join(module_directory, file_name)
