--- a/doc/Makefile
+++ b/doc/Makefile
@@ -6,7 +6,7 @@
 
 # List text files in the order in which you want them to appear in the
 # complete manual:
-SOURCES=README.mdwn INSTALL.mdwn themes.mdwn hacking.mdwn TODO.mdwn modules.mdwn scripts.mdwn faq.mdwn COPYING.mdwn colophon.mdwn
+SOURCES=README.mdwn themes.mdwn hacking.mdwn TODO.mdwn modules.mdwn scripts.mdwn faq.mdwn COPYING.mdwn colophon.mdwn
 OTHER=
 PYTHON_SOURCES:=$(shell find .. -name \*.py)
 TODO_SOURCES=$(patsubst TODO.mdwn,,$(SOURCES)) $(PYTHON_SOURCES)
