--- a/plinth/modules/ikiwiki/views.py
+++ b/plinth/modules/ikiwiki/views.py
@@ -47,7 +47,8 @@
 @login_required
 @package.required(['ikiwiki',
                    'libcgi-formbuilder-perl',
-                   'libcgi-session-perl'],
+                   'libcgi-session-perl',
+                   'libdatetime-format-dateparse-perl'],
                   on_install=on_install)
 def index(request):
     """Serve configuration page."""
