Subject: Include xmpp static files in build.

--- a/setup.py
+++ b/setup.py
@@ -192,6 +192,7 @@
     tests_require=['coverage >= 3.7'],
     include_package_data=True,
     package_data={'plinth': ['templates/*',
+                             'modules/*/static/*',
                              'modules/*/templates/*',
                              'locale/*/LC_MESSAGES/*.[pm]o']},
     data_files=[('/usr/lib/firewalld/services/',
