--- a/setup.py
+++ b/setup.py
@@ -22,6 +22,7 @@
 
 from distutils import log
 from distutils.command.install_data import install_data
+from distutils.util import change_root
 import glob
 import os
 import setuptools
@@ -53,12 +54,14 @@
 
         # Create empty directories
         for directory in DIRECTORIES_TO_CREATE:
+            directory = change_root(self.root, directory)
             if not os.path.exists(directory):
                 log.info("creating directory '%s'", directory)
                 os.makedirs(directory)
 
         # Recursively copy directories
         for target, source in DIRECTORIES_TO_COPY:
+            target = change_root(self.root, target)
             if not os.path.exists(target):
                 log.info("recursive copy '%s' to '%s'", source, target)
                 shutil.copytree(source, target, symlinks=True)
