Description: Correct path to actions scripts
Author: Petter Reinholdtsen <pere@debian.org>

---
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2013-11-17

Index: plinth-alioth/actions.py
===================================================================
--- plinth-alioth.orig/actions.py	2014-05-04 09:41:59.155284983 +0200
+++ plinth-alioth/actions.py	2014-05-04 09:42:00.619298786 +0200
@@ -103,7 +103,7 @@
 
     run_as_root: execute the command through sudo.
     """
-    DIRECTORY = "actions"
+    DIRECTORY = "/usr/share/plinth/actions"
 
     if options == None:
         options = []
