Description: Correct path to actions scripts
Author: Petter Reinholdtsen <pere@debian.org>

---
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2013-11-17

Index: plinth-alioth/actions.py
===================================================================
--- plinth-alioth.orig/actions.py	2013-12-08 19:26:50.570897183 +0100
+++ plinth-alioth/actions.py	2013-12-08 19:45:37.640247133 +0100
@@ -111,7 +111,7 @@
         True # Actions directory hasn't changed.  It's hardcoded :)
 
     """
-    DIRECTORY = "actions"
+    DIRECTORY = "/usr/share/plinth/actions"
 
     if options == None:
         options = []
