Description: Correct URL to issue tracker
Author: Petter Reinholdtsen <pere@hungry.com>

Forwarded: no
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2013-11-17

--- plinth-0.3.0.0.git.20131117.orig/plinth.py
+++ plinth-0.3.0.0.git.20131117/plinth.py
@@ -36,7 +36,7 @@ def error_page(status, dynamic_msg, stoc
 def error_page_404(status, message, traceback, version):
    return error_page(status, message, """<p>If you believe this
    missing page should exist, please file a bug with either the Plinth
-   project (<a href="https://github.com/jvasile/plinth/issues">it has
+   project (<a href="https://github.com/NickDaly/Plinth/issues">it has
    an issue tracker</a>) or the people responsible for the module you
    are trying to access.</p>
 
